### DISCUSSION TITLE
Feature Request: GPT-4V Support

### DISCUSSION THREAD

**Discussion by arpagon** [briansunter/logseq-plugin-gpt3-openai#161] [Ideas]
Hello Brian,

I hope this message finds you well. I've been using your Logseq GPT-3 plugin and it's been incredibly helpful. I wanted to suggest an enhancement that could make it even more valuable: adding support for OpenAI's GPT-4 with vision (GPT-4V).

I believe incorporating this into your plugin could significantly enrich its capabilities.

# A Practical Use Case
Imagine a Logseq user studying or researching, who uploads an image into their notes. With GPT-4V, they could get a detailed AI analysis of this image right in Logseq. This feature would be especially useful for understanding complex diagrams, artworks, or even photographs.

**Comment by TomLucidor** [comment]
Vision models? OpenRouter also cover those, ideally the API should be flexible
